tasks: Add the tasks to build and run locally the production images

andreriesco opened the pull request:
This commit adds the tasks to:
. Build the application based in the  production Dockerfile . Push it to the local registry (localhost:5002)
. Pull it on the board from the local registry
. And run it on the board, showing the result in the VSCode terminal

Commit:
tasks: Add the tasks to build and run locally the production images

This commit adds the tasks to:
. Build the application based in the  production Dockerfile
. Push it to the local registry (localhost:5002)
. Pull it on the board from the local registry
. And run it on the board, showing the result in the VSCode terminal

Signed-off-by: Andre Riesco <andre.riesco@toradex.com>

microhobby commented on assets/tasks/common.json:
```diff
@@ -1518,80 +1438,327 @@
+            ],
+            "problemMatcher": "$msCompile",
+            "icon": {
+                "id": "flame",
```
@andreriesco please check if there is other icon that could be used here, should be more meaningful for the task

Commit:
tasks: Change some icons on run release and cmake clean tasks

Change some icons on run-container-torizon-release-<arch> and
clean-<arch> (cmake template)

Signed-off-by: Andre Riesco <andre.riesco@toradex.com>

microhobby commented:
LGTM
